engine/Logging/logging_config.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+
+MAX_LOG_FILES = 10
```

Comment:
Constants file? Or would it be better here?

Reply:
Since this is not used anywhere else, so I think it would be better here. Also, all logging config is included here.